recipe_scrapers/grandfrais.py

```diff
@@ -0,0 +1,100 @@
+# mypy: allow-untyped-defs
+
+
+import re
+from urllib.parse import urlparse
+
+from ._abstract import AbstractScraper
+from ._utils import FRACTIONS, get_minutes, normalize_string
+
+
+class GrandFrais(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "grandfrais.com"
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return None
+
+    def total_time(self):
+        cook_time, prep_time = self.cook_time(), self.prep_time()
+        if cook_time and prep_time:
+            return cook_time + prep_time
+        return None
+
+    def cook_time(self):
+        raw = [
+            item
+            for item in self.soup.find_all("div", {"class": "pre-requie-item"})
+            if "Cuisson" in item.text
+        ]
+        if raw:
+            return get_minutes(raw[0].text)
+        return None
+
+    def prep_time(self):
+        raw = [
+            item
+            for item in self.soup.find_all("div", {"class": "pre-requie-item"})
+            if "Préparation" in item.text
+        ]
+        if raw:
+            return get_minutes(raw[0].text)
+        return None
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        img = self.soup.find("img", {"alt": self.title()})
+        if img:
+            canonical = urlparse(self.canonical_url())
+            return f"{canonical.scheme}://{canonical.netloc}{img['src']}"
+        return None
+
+    def nutrients(self):
+        return None
+
+    def ingredients(self):
+        # Normalize all ingredients to have a decimal point, remove the prepended
+        # dash, and remove any trailing whitespace.
+        normalize = lambda x: re.sub(
+            r"^(\d+),(\d+)", r"\1.\2", normalize_string(x.lstrip("-").strip())
+        )
+
+        ingredients = [
+            normalize(i)
+            for i in self.soup.find("div", {"itemprop": "ingredients"}).text.split("\n")
+        ]
+
+        for fraction, replacement in FRACTIONS.items():
+            ingredients = [i.replace(fraction, str(replacement)) for i in ingredients]
+
+        return ingredients
+
+    def instructions(self):
+        return "\n".join(
+            [
+                normalize_string(i.text)
+                for i in self.soup.find(
+                    "div", {"itemprop": "recipeInstructions"}
+                ).find_all("ul")
+            ]
+        )
+
+    def ratings(self):
+        return None
+
+    def author(self):
+        return None
```

Comment:
Although the pages don't list the original author(s), as a fallback I think we should provide credit to the business (the copyright holder according to the footer in the page HTML).

```suggestion
    def author(self):
        return "Grand Frais"
```